src/content/doc-surrealdb/installation/upgrading/2.0.mdx

```diff
@@ -7,14 +7,22 @@ description: This guide will help you upgrade your current SurrealDB installatio
 
 # Upgrading to `2.0.0` 
 
-The `2.0.0` release of SurrealDB includes many new features, improvements, and bug fixes. However, because of changes in the underlying way that SurrealDB stores data, three steps are required to migrate to `2.0.0`. We've listed them below, and we advise following this migration guide whether or not you encounter this issue!
+The `2.0.0` release of SurrealDB includes [many new features, improvements, and bug fixes](https://surrealdb.com/releases#v2-0-0). However, because of changes in the underlying way that SurrealDB stores data, three steps are required to migrate to `2.0.0`. 
+
+We've listed them below, and we advise following this migration guide whether or not you encounter this issue!
 
 ## Issues
 
 - A change in the underlying way data is stored in SurrealDB was caused by how ranges and record IDs are stored in the database. 
     -  [Record IDs now support storing UUIDs instead of strings](https://github.com/surrealdb/surrealdb/pull/4491). 
     -  [`Value::Range` stored as just a range instead of a string](https://github.com/surrealdb/surrealdb/pull/4506).
 
+- Some breaking changes were made to the SurrealQL language.
+    -  Behaviour of the `UPDATE` statement which no longer allows to update a record that does not exist. In favor of the [`UPSERT`](/docs/surrealql/statements/upsert) statement.
+    - Deprecated `file://` in favour of `rocksdb://` or the new `surrealkv://`. 
+    - Renamed functions [(meta::*)](/docs/surrealql/functions/database/meta) to (record::*)
+    - Functions `string::endsWith()` and `string::startsWith()` renamed to `string::ends_with()` and `string::starts_with()`
+
 ### Do I have this issue?
```

Comment:
This section confuses me, I don't understand what is meant with "this" issue. This section looks as if it can be removed to me since it doesn't really explain anything

Reply:
renamed it and added a bit more context